content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -440,28 +466,28 @@ spec:
   affinity:
     podAffinity:
       requiredDuringSchedulingIgnoredDuringExecution:
-      # ensure that pods associated with this tenant land on the correct node pool
+      # ensure that Pods associated with this tenant land on the correct node pool
       - matchLabelKeys:
           - tenant
         topologyKey: node-pool
     podAntiAffinity:
       requiredDuringSchedulingIgnoredDuringExecution:
-      # ensure that pods associated with this tenant can't schedule to nodes used for another tenant
+      # ensure that Pods associated with this tenant can't schedule to nodes used for another tenant
       - mismatchLabelKeys:
         - tenant # whatever the value of the "tenant" label for this Pod, prevent
                  # scheduling to nodes in any pool where any Pod from a different
                  # tenant is running.
         labelSelector:
           # We have to have the labelSelector which selects only Pods with the tenant label,
-          # otherwise this Pod would hate Pods from daemonsets as well, for example,
+          # otherwise this Pod would anti-affine Pods from daemonsets as well, for example,
```

Comment:
Oh, that was not a good word for the concept, but affine is not a word, so, maybe this:

```suggestion
          # otherwise this Pod would have anti-affinity against Pods from daemonsets as well, for example,
```

Reply:
Thank you for catching that! You're absolutely right—using "affine" in that context isn't appropriate. I'll update the comment to:
# otherwise this Pod would have anti-affinity against Pods from daemonsets as well, for example,

This change makes the comment clearer and more accurate. I appreciate your suggestion!